archunit/src/main/java/com/tngtech/archunit/core/importer/JavaClassProcessor.java

```diff
@@ -225,6 +227,16 @@ public MethodVisitor visitMethod(int access, String name, String desc, String si
         return result;
     }
 
+    private List<JavaType> typesFrom(String[] exceptions) {
+        List<JavaType> result = new ArrayList<>();
+        if (exceptions != null) {
```

Comment:
This really ever null? I hate those APIs :wink:

Reply:
Yes 😕 See https://asm.ow2.io/javadoc/org/objectweb/asm/ClassVisitor.html#visitMethod-int-java.lang.String-java.lang.String-java.lang.String-java.lang.String:A-

Reply:
Maybe it's for performance reasons, I still hate those API from an user's point of view...